yarn.lock

```diff
@@ -157,23 +157,23 @@
   dependencies:
```

Comment:
I am not sure if we should check-in this file.
@diana-macrometa do you have any idea?

Reply:
If you can add it to the ignore list, do so. I have tried and not managed to.